frontend/src/data/variables/BrfssProvider.ts

```diff
@@ -48,28 +49,25 @@ class BrfssProvider extends VariableProvider {
     let consumedDatasetIds = ["brfss"];
 
     if (breakdowns.geography === "national") {
```

Comment:
It would be good to add a short comment explaining why you set the acs breakdown to "state" if it is "national"